js/scenes/NumberOfGuestScreen/NumberOfGuestScreenContainer.js

```diff
@@ -56,22 +70,66 @@ class NumberOfGuestScreenContainer extends React.Component {
     }
   }
 
-  increase = () => this.setState({ count: this.state.count + 1 });
+  increase = () => this.setState({ count: (this.state.count * 1) + 1 });
```

Comment:
i've changed it to use parseInt() now instead, not sure why * 1 was my goto solution for string to int